src/pages/AddItemPage.tsx

```diff
@@ -0,0 +1,151 @@
+import { useEffect, useState, useRef } from "react";
+import FileInput from "../Component/FileInput";
+import "../../src/style/AddItemPage.css";
+
+interface AddItemState {
+  imgFile: File | null;
+  title: string;
+  description: string;
+  price: string;
+  tag: string[];
+}
+
+function AddItemPage() {
+  const [addItemValue, setAddItemValue] = useState<AddItemState>({
+    imgFile: null,
+    title: "",
+    description: "",
+    price: "",
+    tag: [],
+  });
+
+  const handleChange = (name: string, value: string | File | null) => {
+    setAddItemValue((prevValues) => ({
+      ...prevValues,
+      [name]: value,
+    }));
+  };
```

Comment:
하나로 묶어서 처리하고 싶으셨던것같은데, 이렇게되면 자칫 이상한 값이 상태로 들어갈 수 있어서 버그의 원인이 될수도 있어요ㅠㅠㅠ
그래서 차라리 함수를 나눠서 사용하시는게 좀더 유연할것같아요~

```suggestion
  const onStringChanged = (name: string, value: string) => {
    setAddItemValue((prevValues) => ({
      ...prevValues,
      [name]: value,
    }));
  };
  
const onFileChanged = (name: string, value: File | null) => {
    setAddItemValue((prevValues) => ({
      ...prevValues,
      [name]: value,
    }));
  };
```

그치만 물론 상태값을 나눠서 처리하는게 더 좋긴 해요ㅎㅎ!